### PR TITLE
Fix for hot-spot sprite image not visible on production like environment

### PR DESCRIPTION
**Pull request by ziafazal**
This PR address an issue where hot-spot sprite image not visible when assets are compiled on production environment.  Sprite url was generated with this statement
```
sprite_url = self.runtime.local_resource_url(self, 'public/images/hotspot-sprite.png')
```
which give and invalid url relevant to studio environment some like this (`/c4x/MMBA/IE230/asset/xblock_resources_image_explorer.image_explorer_public_images_hotspot-sprite.8e637638fd95.8e637638fd95.png`)

More detail of the issue can be found [here](https://openedx.atlassian.net/browse/YONK-478)

@bradenmacdonald can some one from you team review this?

### CODE REVIEW

**Comment by bradenmacdonald** [comment]
@ziafazal Sure thing, we'll assign a reviewer on Monday.

**Comment by ziafazal** [comment]
@e-kolpakov Can you review this?

**Comment by itsjeyd** [comment]
@ziafazal I'll be reviewing this PR for our team.

File: image_explorer/image_explorer.py [hunk 1]
```diff
@@ -99,20 +99,17 @@ def student_view(self, context):
...
             'background': background,
-            'sprite_url': sprite_url,
         }
 
         fragment = Fragment()
         fragment.add_content(loader.render_template('/templates/html/image_explorer.html', context))
-        fragment.add_css(loader.load_unicode('public/css/image_explorer.css'))
-        fragment.add_javascript(loader.load_unicode('public/js/image_explorer.js'))
+        fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/image_explorer.css'))
+        fragment.add_javascript_url(self.runtime.local_resource_url(self, 'public/js/image_explorer.js'))
```
Comment (itsjeyd) [inline image_explorer/image_explorer.py]: @ziafazal Just to confirm, the changes in L111, L112, and L176 are drive-by changes, right?

Comment (ziafazal) [inline image_explorer/image_explorer.py]: @itsjeyd L112 and L176 are drive-by changes but L111 is need to fix the issue. `fragment.add_css_url ` method adds xblock resource by url whereas `fragment.add_css` adds resource as inline piece of code and inline css is not picked up by asset pipeline.

Comment (itsjeyd) [inline image_explorer/image_explorer.py]: @ziafazal OK, thanks for clarifying.
